server/src/main.rs

```diff
@@ -80,6 +80,7 @@ async fn main() -> Result<(), anyhow::Error> {
     let token_validator_schedule = token_validator.clone();
     let lazy_feature_cache = features_cache.clone();
     let lazy_token_cache = token_cache.clone();
+    // TODO : do we need lazy delta cache?
```

Comment:
Nope, if it compiles then you don't need it. The lazy stuff is to handle cases where something is used more than once later in this function and we need a reference

Might need this if we add this to the persistent store layer

```suggestion
```